pages/api/roles/index.ts

```diff
@@ -68,12 +69,35 @@ function handler(req: NextApiRequest, res: NextApiResponse) {
   }
 }
 
-// TODO: Optional Search query. Will be done in another PR to unblock others.
-
 async function handleGet(req: NextApiRequest, res: NextApiResponse<ApiResponse<RoleListData[]>>) {
```

Comment:
Also, could you merge #35 to fix the handlers in this PR (and elsewhere)

Reply:
Thanks!